### ISSUE TITLE
Visionary 5 Series products

### ISSUE THREAD

**Issue by Madcap-Magician** [bitfocus/companion-module-requests#1587]
Please add in support for visionary-av products. 

I have it working with generic UDP, but integrated support would be amazing

5 series encoders and decoders share common API and use UDP on port 8000

Example string: CMD=START&UNIT.ID=ALL&STREAM.HOST=192.168.8.101&STREAM.CONNECT=T
RUE&VW.ACTIVE=FALSE&CMD=END

Full API Documentation attached.

Can provide anymore detail or info needed needed including more technical documentation as needed.

[25-00186-042-X5K_API_Specification_3_x-2.pdf](https://github.com/user-attachments/files/16656697/25-00186-042-X5K_API_Specification_3_x-2.pdf)

**Comment by pateboyd** [comment]
Issue 828 is also requesting but for version 4. I believe the API is the same. 

But, this module would be useful for me too.

**Comment by Madcap-Magician** [comment]
> Issue 828 is also requesting but for version 4. I believe the API is the same.
> 
> But, this module would be useful for me too.

Yep, mostly shared, some things wont work with the 4 series like breakaway usb/aud/vid and some of the new features, but switching would be fine

**Comment by poshlivin** [comment]
This module would be a game changer for efficient routing for visionary components.

**Comment by AVDOXA1** [comment]
I would love to see this module!
